src/PKSim.Presentation/Presenters/Individuals/IndividualTransporterExpressionsPresenter.cs

```diff
@@ -90,6 +90,8 @@ public void SetMembraneLocation(TransporterExpressionContainerDTO transporterExp
       public void UpdateTransportType(TransportType newTransportType)
       {
          AddCommand(_moleculeExpressionTask.SetTransporterTypeFor(_transporter, newTransportType));
+         //required to refresh view when changing transport type as some UI elements need to be invalidated
+         RefreshView();         
```

Comment:
When transporter type is changing, some of the controls are in an outdated state (list of selected membrane etc.. do not match). This way, we ensure that everything is reset